cypher/cypher-docs/src/docs/dev/deprecations-additions-and-compatibility.asciidoc

```diff
@@ -40,6 +40,8 @@ Replacement syntax for deprecated and removed features are also indicated.
 | `CREATE CONSTRAINT [name] ON ()-[rel:REL]-() ASSERT exists(rel.property)` | Syntax | Deprecated | Replaced by `CREATE CONSTRAINT [name] ON ()-[rel:REL]-() ASSERT rel.property IS NOT NULL`
 | `exists(prop)` | Syntax | Deprecated | Replaced by `prop IS NOT NULL`
 | `NOT exists(prop)` | Syntax | Deprecated | Replaced by `prop IS NULL`
+| `BRIEF [OUTPUT]` for `SHOW INDEXES` and `SHOW CONSTRAINTS`   | Syntax | Deprecated | Replaced by default output columns
+| `VERBOSE [OUTPUT]` for `SHOW INDEXES` and `SHOW CONSTRAINTS` | Syntax | Deprecated | Replaced by `YIELD *`
```

Comment:
`YIELD` is not yet implemented, either update description or block until `YIELD` exists for the show schema commands